linter/linter.go

```diff
@@ -69,14 +70,19 @@ func New(realIssuer *x509.Certificate, realSigner crypto.Signer, skipLints []str
 
 // Check signs the given TBS certificate using the Linter's fake issuer cert and
 // private key, then runs the resulting certificate through all non-filtered
-// lints. It returns an error if any lint fails.
-func (l Linter) Check(tbs *x509.Certificate, subjectPubKey crypto.PublicKey) error {
-	cert, err := makeLintCert(tbs, subjectPubKey, l.issuer, l.signer)
+// lints. It returns an error if any lint fails. On success it also returns the
+// DER bytes of the linting certificate.
```

Comment:
Since we're returning and passing around raw DER bytes, rather than an x509 object, any interest in having those bytes be just the tbsCertificate?

Reply:
Interest, yes. Capability, no. While we can extract the RawTBSCertificate bytes no problem, nothing in the Go stdlib allows us to subsequently _parse_ tbsCertificate bytes, only Certificate bytes.

Reply:
Ah right, so we could toss it into the database just fine, but be stuck handling it as opaque bytes from then on. Fair enough!